tests/check_framework/urls/good_handlers.py

```diff
@@ -0,0 +1,14 @@
+urlpatterns = []
+
+handler400 = __name__ + '.good_handler'
+handler403 = __name__ + '.good_handler'
+handler404 = __name__ + '.good_handler'
+handler500 = __name__ + '.good_handler500'
```

Comment:
curious why this line is being handled different than the `bad_handler` equivalent?

Reply:
`handler500` takes a different number of arguments - only one. But yes a view could be used with optional `exception` argument, I'll do that.